pages/your_tasks/how_to_make_a_good_readme.md

```diff
@@ -0,0 +1,125 @@
+---
+title: How to make a good README
+search_exclude: true
+description: Description of good practice to make a good README for research software.
+contributors: ["Esteban González"]
+page_id: good_README
+related_pages: []
+# More information on which page id you can use can be found at https://rdmkit.elixir-europe.org/website_overview
+training:
+  - name:
+    registry:
+    url:
+# More information on how to fill in this metadata section can be found here https://rdmkit.elixir-europe.org/page_metadata
+---
+<!-- Please take in mind our style guide https://rdmkit.elixir-europe.org/style_guide when writing the content of this page. -->
+
+## Why is it important a good README file? <!-- example: how to version control my code? -->
+ 
+### Description <!-- do not delete this heading and write your text below it -->
+
+The README file is the entry point to the documentation associated with a software. The main purpose of the README is:
+* Describe the structure of the project and its purpose.
+* Facilitate the installation and execution of the software stored in the repository.
+* Detail the use of the software.
```

Comment:
Point to the software's licence for the terms of reuse?

Reply:
I will merge for now, and we can do another review